core/data_sanitizer.py

```
@@ -0,0 +1,346 @@
+- Safe defaults for missing or invalid values
+"""
+
+from __future__ import annotations
+
+import logging
+import math
+from typing import Any
+
+from pydantic import BaseModel, Field, field_validator
+
+from core.nutrition_constants import (
```

Flake8 (7.3.0)

[error] 'core.nutrition_constants.CARBS_G_MIN' imported but unused

(F401)